### PR TITLE
Moved price processing methods nearer to usage + fixed a potential bu…

### PR DESCRIPTION
**Pull request by mingazhev**
Moved functions nearer to usage + fixed invalid markup calculation on different currencies

### CODE REVIEW

File: Api/Services/Accommodations/Availability/RoomContractSetPriceProcessing.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using HappyTravel.Edo.Api.Services.PriceProcessing;
+using HappyTravel.EdoContracts.Accommodations.Internals;
+using HappyTravel.EdoContracts.General;
+using HappyTravel.Money.Models;
+
+namespace HappyTravel.Edo.Api.Services.Accommodations.Availability
+{
+    public static class RoomContractSetPriceProcessing
```
Comment (hasmikharutyunyan) [inline Api/Services/Accommodations/Availability/RoomContractSetPriceProcessing.cs:10]: Maybe `RoomContractSetPriceProcessingService` or `RoomContractSetPriceProcessor` ?

File: Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationPriceProcessing.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using HappyTravel.Edo.Api.Services.PriceProcessing;
+using HappyTravel.Money.Enums;
+
+namespace HappyTravel.Edo.Api.Services.Accommodations.Availability.Steps.BookingEvaluation
+{
+    public static class BookingEvaluationPriceProcessing
```
Comment (hasmikharutyunyan) [inline Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationPriceProcessing.cs:7]: Same here?

File: Api/Services/Accommodations/Availability/RoomContractSetPriceProcessing.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Availability/RoomContractSetPriceProcessing.cs]: Changed to instance class and renamed

File: Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationPriceProcessing.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationPriceProcessing.cs]: Changed to instance class and renamed here, and in other places

**Review by kirillta — APPROVED** [review]

**Review by hasmikharutyunyan — APPROVED** [review]

**Pull request merged**